FreestarOBJCSample/FreestarOBJCSample/ThumbnailViewController.m

```diff
@@ -0,0 +1,71 @@
+//
+//  ThumbnailViewController.m
+//
+//  Created by Carlos Alcala on 9/02/22.
+//  Copyright © 2019. All rights reserved.
+//
+
+#import "ThumbnailViewController.h"
+#import <UIKit/UIKit.h>
+#import <FreestarAds/Freestar.h>
+#import <FreestarAds/FreestarThumbnailAd.h>
+
+
+@interface ThumbnailViewController()<FreestarThumbnailAdDelegate>
+
+@property (nonatomic, weak) IBOutlet UITextView *statusTextView;
+@property (nonatomic, strong) FreestarThumbnailAd *thumbnailAd;
+@property (nonatomic, assign) BOOL thumbnailAdReady;
+
+@end
+
+@implementation ThumbnailViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+
+    [self setupConfig];
+
+}
+
+- (void)setupConfig {
+    [FreestarThumbnailAd setWhitelistBundleIdentifiers:@[@"io.freestar.ads.FreestarSwiftSample"]];
+    [FreestarThumbnailAd setBlacklistViewControllers:@[@"BlackViewController",@"FullscreenAdViewControllerPush",@"FullscreenAdViewController"]];
+    [FreestarThumbnailAd setGravity:TopLeft];
+    [FreestarThumbnailAd setXMargin:100];
+    [FreestarThumbnailAd setYMargin:500];
+}
+
+-(void)loadThumbnailAd {
+    self.thumbnailAd = [[FreestarThumbnailAd alloc] init];
+    self.thumbnailAd.delegate = self;
+    [self.thumbnailAd load];
```

Comment:
This is optional, but you may want to think about adding a textfield so the user can enter in their own placement id.